corehq/apps/locations/models.py

```diff
@@ -438,7 +438,8 @@ def __init__(self, *args, **kwargs):
             self.location_type = location_type
 
     def __repr__(self):
-        return "%s (%s)" % (self.name, self.location_type)
+        val = u"%s (%s)" % (self.name, self.location_type)
+        return val.encode('utf-8')
```

Comment:
@gcapalbo Any tips on where I can find this being called?  I'd like to provide a more helpful dev-facing representation (which includes the type of the object), but I wanna make sure I don't also make that user-facing.